schema/status.py

```diff
@@ -0,0 +1,123 @@
+status_p = {
+    "character_id": "s",
+    "text_1" : "t",
+    "texture" : "t",
+    "model": "t",
+    "float_1": "f",
+    "float_2": "f",
+    "float_3": "f",
+    "float_4": "f",
+    "float_5": "f",
+    "float_6": "f",
+    "float_7": "f",
+    "short_1": "s",
+    "short_2": "s",
+    "is_female": "b",
+    "level": "b",
+    "hp_base": "i",
+    "hp_growth": "f",
+    "max_ep": "s",
+    "start_ep": "s",
+    "max_cp": "s",
+    "start_cp": "s",
+    "str_base": "s",
+    "str_growth": "f",
+    "def_base": "s",
+    "def_growth": "f",
+    "ats_base": "s",
+    "ats_growth": "f",
+    "adf_base": "s",
+    "adf_growth": "f",
+    "dex_base": "s",
+    "dex_growth": "f",
+    "agi_base": "s",
+    "agi_growth": "f",
+    "eva": "s",
+    "spd_base": "s",
+    "spd_growth": "f",
+    "mov_base": "s",
+    "mov_growth": "f",
+
+    "exp_base": "s",
+    "exp_growth": "f",
+    "break_multiplier_base": "s",
+    "break_multiplier_growth": "f",
+    "earth_efficacy" : "b",
+    "water_efficacy" : "b",
+    "fire_efficacy" : "b",
+    "wind_efficacy" : "b",
+    "time_efficacy" : "b",
+    "space_efficacy" : "b",
+    "mirage_efficacy" : "b",
+
+    "psn_efficacy" : "b",
+    "seal_efficacy" : "b",
+    "mute_efficacy" : "b",
+    "blnd_efficacy" : "b",
+    "slp_efficacy" : "b",
+    "burn_efficacy" : "b",
+    "frz_efficacy" : "b",
+    "petr_efficacy" : "b",
+    "fnt_efficacy" : "b",
+    "conf_efficacy" : "b",
+    "chrm_efficacy" : "b",
+    "dblw_efficacy" : "b",
+    "nmr_efficacy" : "b",
+    "dlay_efficacy" : "b",
+    "vnsh_efficacy" : "b",
+    "s-dwn_efficacy" : "b",
+
+    "slash_efficacy" : "s",
+    "thurst_efficacy" : "s",
+    "pierce_efficacy" : "s",
+    "strike_efficacy" : "s",
+
+    "zeros_1": "d46",
```

Comment:
Can you check if all of these are just 0s? If that is not the case, then you might want to compare to the schema of t_mons to check

Reply:
Yes, zeros_1 is all zero bytes in every row. The corresponding fields in t_mons are sepith/item drops and some floats, so they're probably unused here. (I didn't check though)